kore/test/Test/Kore/Builtin/Definition.hs

```diff
@@ -696,6 +696,10 @@ string2IntStringSymbol :: Internal.Symbol
 string2IntStringSymbol =
     builtinSymbol "string2intString" intSort [stringSort]
         & hook "STRING.string2int"
+base2StringStringSymbol :: Internal.Symbol
```

Comment:
This needs to be added to the test (Kore) module; look for `stringModule` below and follow the example of `string2baseStringSymbol`.